integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/GlobalDictionaryUtil.scala

```diff
@@ -182,7 +182,7 @@ object GlobalDictionaryUtil {
       case None =>
         None
       case Some(dim) =>
-        if (DataTypes.isArrayType(dim.getDataType)) {
+        if (DataTypes.isArrayType(dim.getDataType) || DataTypes.isMapType(dim.getDataType)) {
```

Comment:
Does mapType also use ArrayParser ?

Reply:
Yes, Map is implemented as Array of Struct internally.